.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - run:
           name: PHP 5.5 Magento 1.8
           command: |
-            PHPUNIT_PHAR=tests/unit/phpunit-4.8.36.phar MAGENTO_VERSION=magento-mirror-1.8.1.0 tests/scripts/ci.sh
+            PHPUNIT_PHAR=tests/integration/phpunit-4.8.36.phar MAGENTO_VERSION=magento-mirror-1.8.1.0 tests/scripts/ci.sh
```

Comment:
As @daisy1754  mentioned, these are unit test, concerned with individual units of code and not concerned with each units integration with the other pieces of the code.  Integration is concerned with inter-dependencies, unit test are not.

Reply:
@mccottry , @daisy1754 you are right - in all documentation about Magento 1.* plugins such tests are called "unit". Really, integration testing is the phase in software testing in which individual software modules are combined and tested as a group. In order to run that "unit" tests we need to install Magento.
Unit testing means that we could test something without installing Magento

Reply:
Great @oldDriver .  Thanks for clarifying.  

For our purposes, we use PHP*Unit* here in are unit testing.  Also, we use the common OO perspective that the unit of focus will be our class methods that do require the PHP libraries but do not require that the methods are run in the context of the full application.  

So, @daisy1754 is requesting that we not make the change in name from `unit` to `integrataion` and I definitely fully agree.  There are already true integration test defined elsewhere and this change will make things a little bit confusing for others.

Reply:
@mccottry @daisy1754 Could you propose name for real unit tests which could be run without Magento integration.
I could not put new unit test in the one folder with "unit/integration" tests, because they have different boostrap.
I could restore folder name to "unit" for integration tests, and put real unit tests into another folder. But i have no edea how to name it. May be "local", does it make sense?

Reply:
@oldDriver Please try 
1.) restoring the folder name to `unit` and keeping all PHP*Unit* test there.  
2.) Place any new integration test in a separate folder.

When you write new unit or integration test, you can demonstrate how you visualize pure unit test and also proper integration test should be written.

To me, it seems like you are wanting to challenge the validity of our current definition of unit test and integration test, which is that: 
- The units that we are interested in testing are the individual methods which may use standard Magento libraries and DB resources.  
- Integration test are concerned interactions between Bolt, web browser, DBs, Magento, and plugin code in near to true life context

I certainly understand that you would like that all Magento and DB dependencies be stubbed and mocked in unit test.  This point of view is also valid.  I suggest to pursue this, you should open this as a separate issue and present your proposal in Integration Weekly.  Another valid approach to your proposed change is to go test by test and stub out and mock out all DB and Magento dependencies so that they fit your desired definition of unit test.  You will then make PRs for each of these changes.

Hopefully, you can understand what I written above.  If not, feel free to give me a call and I can elaborate.